### DISCUSSION TITLE
Export the data as a CSV

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#585] [Feature requests]
From the index view have an export button.

Maybe using the plugin system.

**Comment by benjaminb10** [comment]
A custom export CSV letting final user choosing which column he wants can be a great solution. Just like on Stripe:

![image](https://user-images.githubusercontent.com/2952678/200328405-80da0581-dbe4-490a-b68b-66285ca969a0.png)

**Reply by adrianthedev** [reply]
Yes, I know what you mean. That's a different (rather heavy) feature we'd love to bring.

You can, however, create your own version using actions. [Here's](https://github.com/avo-hq/avo/pull/1368/files#diff-717fe3544d6ab7ae77c9838e08f9410d5a244c6dab09b23a422b05a2efb73234) a sample of that.

**Reply by adrianthedev** [reply]
I posted an updated example of this on our docs page. It even supports custom field selection.

https://docs.avohq.io/2.0/recipes/export-to-csv.html
